packages/pouchdb-core/src/adapter.js

```diff
@@ -620,23 +619,22 @@ AbstractPouchDB.prototype.get =
       var attachments = doc._attachments;
       var count = Object.keys(attachments).length;
       if (count === 0) {
-        return callback(null, doc);
+        return cb(null, doc);
       }
       Object.keys(attachments).forEach(function (key) {
         this._getAttachment(doc._id, key, attachments[key], {
           // Previously the revision handling was done in adapter.js
           // getAttachment, however since idb-next doesnt we need to
           // pass the rev through
           rev: doc._rev,
-          binary: opts.binary,
-          ctx: ctx
```

Comment:
We attempted to pass `ctx` here, however the transaction cant survive, there is a `forEach` above that the transaction may survive over (forgot the exact rules), however https://github.com/pouchdb/pouchdb/blob/master/packages/pouchdb-adapter-idb/src/index.js#L381 will definitely break it, it chrome but firefox is stricter about that and timing will effect it, depending on it is certainly broken